blazar_dashboard/api/client.py

```diff
@@ -363,6 +364,31 @@ def extra_capability_names(request):
     return list(set(available))
 
 
+def computehost_extra_capabilities_values(request, name):
+    """
+    Return the capabilities with a given "name". The client can cache/combine
+    the rows together to build up a full copy of the extra capabilities table
+    if they really want. Then they can do the "what-if" filtering themselves
+    to count hosts.
+
+    They could maybe mix that with the calendar data to even see if the chosen
+    number of hosts are free. Might need to do the lookup between
+    computehost_id (small integers) and the UUID via the uid name:value pairs.
+    """
+    cursor = get_cursor_for_request(request)
+    sql = '''\
+    SELECT
+        id, computehost_id, capability_name, capability_value
+    FROM
+        computehost_extra_capabilities
+    WHERE
+        capability_name = %s
+    '''
+    cursor.execute(sql, [name])
```

Comment:
This is getting us away from our goal of reducing DB access. Can't we get this from the Blazar host list?

Reply:
Err, I see that this isn't even called. What is it for?

Reply:
Meant to remove that. Played with that first, but can do with host_list. Updates pushed.